pandas/core/arrays/categorical.py

```diff
@@ -2504,15 +2508,15 @@ def codes(self) -> Series:
         """
         from pandas import Series
 
-        return Series(self._parent.codes, index=self._index)
+        return Series(self._parent.codes, index=self._index, copy=False)
```

Comment:
This should maybe not be a copy=False? (modifying the series should never mutate those codes)

Reply:
Good point